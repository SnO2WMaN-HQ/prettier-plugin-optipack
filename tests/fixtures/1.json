{
  "name": "@optipack/root",
  "private": true,
  "description": "Optimizer for package.json",
  "license": "MIT",
  "author": {
    "email": "me@sno2wman.dev",
    "name": "SnO2WMaN"
  },
  "workspaces": ["@optipack/*"],
  "scripts": {
    "test": "lerna run test --stream --parallel --include-dependencies",
    "test:ci": "lerna run test:ci --stream --parallel --include-dependencies",
    "lint:eslint": "eslint . --ext .js,.ts",
    "lint:prettier": "prettier --check **/*.{json,yml,yaml,md}",
    "build": "lerna run build --stream --parallel --include-dependencies",
    "clean": "run-s -cn clean:*",
    "clean:git": "lerna exec 'git clean -xdf node_modules lib coverage tsconfig.tsbuildinfo'",
    "clean:lerna": "lerna clean --yes",
    "fmt": "run-s -cn fmt:*",
    "fmt:eslint": "yarn lint:eslint --fix",
    "fmt:prettier": "yarn lint:prettier --write",
    "lint": "run-s -cn lint:*"
  },
  "devDependencies": {
    "@arkweid/lefthook": "0.7.2",
    "@commitlint/cli": "11.0.0",
    "@commitlint/config-conventional": "11.0.0",
    "@commitlint/config-lerna-scopes": "11.0.0",
    "@shopify/eslint-plugin": "38.0.0",
    "@shopify/prettier-config": "1.1.1",
    "@types/jest": "26.0.13",
    "@types/lodash": "4.14.161",
    "@types/prettier": "2.1.0",
    "conventional-changelog-cli": "2.1.0",
    "eslint": "7.8.1",
    "jest": "26.4.2",
    "lerna": "3.22.1",
    "lodash": "4.17.20",
    "npm-run-all": "4.1.5",
    "prettier": "2.1.1",
    "prettier-plugin-organize-imports": "1.1.1",
    "ts-jest": "26.3.0",
    "ts-node": "9.0.0",
    "tsconfig-paths": "3.9.0",
    "typescript": "3.9.7"
  },
  "dependencies": {
    "@optipack/cli": "file:@optipack/cli",
    "@optipack/core": "file:@optipack/core",
    "@optipack/prettier-plugin-optipack": "file:@optipack/prettier-plugin"
  }
}
