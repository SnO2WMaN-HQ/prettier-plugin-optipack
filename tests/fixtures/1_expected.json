{
  "name": "@optipack/root",
  "private": true,
  "description": "Optimizer for package.json",
  "license": "MIT",
  "author": {
    "email": "me@sno2wman.dev",
    "name": "SnO2WMaN"
  },
  "workspaces": [
    "@optipack/*"
  ],
  "scripts": {
    "build": "lerna run build --stream --parallel --include-dependencies",
    "test": "lerna run test --stream --parallel --include-dependencies",
    "test:ci": "lerna run test:ci --stream --parallel --include-dependencies",
    "clean": "run-s -cn clean:*",
    "clean:git": "lerna exec 'git clean -xdf node_modules lib coverage tsconfig.tsbuildinfo'",
    "clean:lerna": "lerna clean --yes",
    "lint": "run-s -cn lint:*",
    "lint:eslint": "eslint . --ext .js,.ts",
    "lint:prettier": "prettier --check **/*.{json,yml,yaml,md}",
    "fmt": "run-s -cn fmt:*",
    "fmt:eslint": "yarn lint:eslint --fix",
    "fmt:prettier": "yarn lint:prettier --write"
  },
  "dependencies": {
    "@optipack/cli": "file:@optipack/cli",
    "@optipack/core": "file:@optipack/core",
    "@optipack/prettier-plugin-optipack": "file:@optipack/prettier-plugin"
  },
  "devDependencies": {
    "@arkweid/lefthook": "0.7.2",
    "@commitlint/cli": "11.0.0",
    "@commitlint/config-conventional": "11.0.0",
    "@commitlint/config-lerna-scopes": "11.0.0",
    "@shopify/eslint-plugin": "38.0.0",
    "@shopify/prettier-config": "1.1.1",
    "@types/jest": "26.0.13",
    "@types/lodash": "4.14.161",
    "@types/prettier": "2.1.0",
    "conventional-changelog-cli": "2.1.0",
    "eslint": "7.8.1",
    "jest": "26.4.2",
    "lerna": "3.22.1",
    "lodash": "4.17.20",
    "npm-run-all": "4.1.5",
    "prettier": "2.1.1",
    "prettier-plugin-organize-imports": "1.1.1",
    "ts-jest": "26.3.0",
    "ts-node": "9.0.0",
    "tsconfig-paths": "3.9.0",
    "typescript": "3.9.7"
  }
}
